.changeset/seven-rivers-greet.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
Changeset for a new minor of `@clerk/shared` to highlight that new helper 👍

Reply:
changeset updated!